packages/react-accordion/src/components/AccordionItem/AccordionItem.types.ts

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
-export type AccordionItemContextValue = Omit<AccordionItemCommons, 'value'> & {
-  open: boolean;
+export type AccordionItemContextValue = {
+  disabled: boolean;
```

Comment:
This should probably still be in terms of `AccordionItemProps`:

```suggestion
export type AccordionItemContextValue = Required<Pick<AccordionItemProps, disabled>> & {
```

Reply:
haha I actually wrote that first, stared at it for about 10s, and then removed it :D.

I was primarily going back and forth on whether the ContextValue was conceptually tied to props, or its own thing. I'm fine switching it back though :)